package/MDAnalysis/lib/transformations.py

```diff
@@ -109,6 +109,7 @@
 Examples
 --------
 
+>>> from MDAnalysis.lib.transformations import identity_matrix
```

Comment:
What about the other functions such as `rotation_matrix`, `euler_from_matrix`, ...? (Same comment applies elsewhere).

Reply:
Hmm, I really didn't take that in mind. I will import them too.

Reply:
@AHMED-salah00 I took me a while to figure this out, but https://github.com/MDAnalysis/mdanalysis/pull/3997 added the doctests to CI. These never fail, so you need to dig them up yourself and see. 

For this PR, you can go under the "Checks" tab, look for the "build_docs" test under the "GH Actions CI" menu on the left, and then look at (expand) the "doctest" phase. If you search for `lib/transformations.py` you can see the current failures (due to missing `rotation_matrix`, etc.). You can do the same for the other files you touched in this PR and hopefully fix all the current issues.

@MDAnalysis/coredevs I think here it would be more useful here to have CI tests that can be incrementally turned on (instead of a single CI test for everything that never fails) when a file is addressed and fixed, so that we can ensure a PR is OK (by turning on the checks for the current files) and more importantly that no more issues are re-introduced later on. Would this be possible with doctests?

Reply:
Not easily afaik - at least not with the current way we're invoking things (coming up with more complex solutions might take a while). I think the original suggestion was for reviewers to compare against the last merge commit and see if the numbers went up or down.

Reply:
Thanks @RMeli for this found lots of failing docs here :)

Reply:
Indeed. =) Please focus only on one or a few files at a time, to keep PRs of reasonable size (thus easier to review and merge). For the current PR I'd fix all the issues in the files you already touched:
* `core/universe.py`
* `lib/transformations.py`
* `lib/util.py`